internal/pkg/db/redis/queries.go

```diff
@@ -291,7 +291,7 @@ func getObjectsByValuesSorted(conn redis.Conn, limit int, vals ...string) (objec
 		return nil, err
 	}
 
```

Comment:
Here should check the len(ids) ==0 to prevent the same error `ERR wrong number of arguments for 'mget' command`
```	
	if len(ids) ==0 {
		return objects ,nil
	}
```